lib/redis/partition_cache.ex

```diff
@@ -1,12 +1,14 @@
 defmodule Castle.Redis.PartitionCache do
+  use Memoize
+
   alias Castle.Redis.Conn, as: Conn
 
   def partition(key, combiner_fn, worker_fns) do
     get_parts(key, 0, nil, worker_fns) |> combine(combiner_fn)
   end
   def partition(key, fns), do: partition(key, fn(parts) -> parts end, fns)
 
-  def partition_get(key, num_parts, combiner_fn) do
+  defmemo partition_get(key, num_parts, combiner_fn), expires_in: 300 * 1000 do
```

Comment:
@gcampo88 so I was finding that querying `/api/v1/podcasts/1234` locally was actually kind of slow (considering everything is cached).  It was because those rollup redis keys are just so dang big.  On my local machine it was something like:

```
185.24 Kb  rollup.trends.0
146.13 Kb  rollup.trends.1
149.21 Kb  rollup.totals.0
146.20 Kb  rollup.totals.1
134.07 Kb  rollup.totals.2
```

Not an ideal use case for redis.  But rather than fix the problem with how rollups work, I just threw in a 5-minute memoization here.  And I opened #54.

Reply:
👍

Reply:
yeah dang. those redis values are huge! took Atom a second just to paste one in